Randomizer.cs

```diff
@@ -1195,7 +1195,7 @@ private void PlaceFreeItems(List<Item> itemPool)
                 Item.StartingHeartContainer1,
                 Item.StartingHeartContainer2,
             };
-            var availableStartingItems = (_settings.NoStartingItems
+            var availableStartingItems = (_settings.RandomStartingItems
                 ? ItemUtils.AllRupees()
                 : ItemUtils.StartingItems())
```

Comment:
These are reversed, please fix them.

Reply:
Reverse?

Reply:
you reversed the `if` condition, so now you need to swap the `then` and the `else`

Reply:
Fixed.